Userland/Libraries/LibJS/Tests/classes/class-constructor.js

```diff
@@ -65,3 +65,10 @@ test("implicit constructor", () => {
 
     expect(new A()).toBeInstanceOf(A);
 });
+
+test("can call constructor with parenthesis", () => {
```

Comment:
```suggestion
test("can call constructor without parenthesis", () => {
```

?

Reply:
While we're at it, I think it should say `parentheses`,

Reply:
Whoops, thanks.